src/NzbDrone.Core/Download/CompletedDownloadService.cs

```diff
@@ -80,8 +80,12 @@ public void Process(TrackedDownload trackedDownload, bool ignoreWarnings = false
 
                 if (series == null)
                 {
-                    trackedDownload.Warn("Series title mismatch, automatic import is not possible.");
-                    return;
+                    series = _parsingService.GetSeries(historyItem.SourceTitle);
```

Comment:
`historyItem` could be null here if the download wasn't initiated by sonnar, you need to do a check before trying to get it's `SourceTitle`